pkg/controller/master/image/backing_image_controller.go

```diff
@@ -44,17 +45,20 @@ func (h *backingImageHandler) OnChanged(_ string, backingImage *lhv1beta1.Backin
 			harvesterv1beta1.ImageImported.False(toUpdate)
 			harvesterv1beta1.ImageImported.Reason(toUpdate, "ImportFailed")
 			harvesterv1beta1.ImageImported.Message(toUpdate, status.Message)
+			harvesterv1beta1.ImageImported.LastUpdated(toUpdate, time.Now().Format(time.RFC3339))
```

Comment:
Just to confirm, are these TS updates handling the cases `False/Reason/Message` not changes (since they only touch the TS if values changes)?

Reply:
Yes, [wrangler](https://github.com/rancher/wrangler/blob/ce552e66572040485844070ca1ce8323f48ded39/pkg/condition/condition.go#L167-L173) doesn't update TS if the value is not changed. We use `lastUpadteTime` in the condition to determine whether we've waited for 10 seconds and whether we can download the image again.